src/core/plugin/plugin.service.js

```diff
@@ -1,4 +1,4 @@
-import {AppError} from '../infrastructure';
+import { AppError } from 'ng2-qgrid/core/infrastructure/error';
```

Comment:
in core it's restricted to use `ng2-qgrid` because core can be used in ng-qgrid too
use relative path